tests/foreman/ui_airgun/test_computeresource.py

```diff
@@ -0,0 +1,76 @@
+from robottelo.datafactory import gen_string, valid_data_list
+from robottelo.decorators import parametrize
+from robottelo.config import settings
+
+
+@parametrize('name', valid_data_list())
```

Comment:
Please use `**valid_data_list('ui')` instead. Without specifying interface it will generate SMP chars for utf8, which are not supported by chrome and tests will fail.